contrib/node/src/python/pants/contrib/node/tasks/node_bundle.py

```diff
@@ -0,0 +1,61 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+
+from pants.base.build_environment import get_buildroot
+from pants.fs import archive
+
+from pants.contrib.node.tasks.node_paths import NodePaths
+from pants.contrib.node.tasks.node_task import NodeTask
+
+
+class NodeBundle(NodeTask):
+  """Create an archive bundle of NodeModule targets."""
+
+  @classmethod
+  def register_options(cls, register):
+    super(NodeBundle, cls).register_options(register)
+    # Some node modules depend on symlink to function.  Thus we can only support archival type
+    # that can preserve symlinks.
+    register('--archive', choices=list(archive.TYPE_NAMES_PRESERVE_SYMLINKS),
```

Comment:
I gave it some thoughts and an additional target node_bundle does not make much sense currently.  The suggested node_bundle will always have only one dependency which is a node_module since archive two node modules as sibling directories is not a valid/common scenario.  Maybe we just put the archive options in the node_module target itself?  Suggestions are welcome.